### PR TITLE
Make sure the hostname is configured with a FQDN

### PR DESCRIPTION
**Pull request by kbidarkar**

### CODE REVIEW

**Comment by kbidarkar** [comment]
Fixed in https://github.com/SatelliteQE/automation-tools/issues/105

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -244,6 +244,9 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
     if len(hostname) == 0:
         print('Was not possible to fetch hostname information')
         sys.exit(1)
+    if '.' not in hostname:
```
Comment (rplevka) [inline automation_tools/__init__.py]: perhaps we could merge the checks and have it like:
```python
if '.' not in hostname or len(hostname.split('.', 1)[1]) == 0:
    print('Please make sure the hostname is configured with a valid FQDN')
    sys.exit(1)
```

Comment (elyezer) [inline automation_tools/__init__.py]: Better to have separated otherwise will be required to repeat `hostname.split('.', 1)[1]`

**Comment by elyezer** [comment]
ACK

**Comment by sthirugn** [comment]
ACK

**Comment by sthirugn** [comment]
I will let @rplevka review again and merge.

**Comment by rplevka** [comment]
@sthirugn, thanks :)
ACK

**Pull request merged**